Leicester/Leicester210.xml

```diff
@@ -329,6 +328,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
          <change who="VP" when="2022-09-26">Created catalogue entry stub</change>
       </revisionDesc>
    </teiHeader>
+   <facsimile>
+<graphic url="https://specialcollections.le.ac.uk/iiif/info/p15407coll6/20030/manifest.json"></graphic>
```

Comment:
please here give the URI, not the manifest )

Reply:
(I know it is confusing. facs takes manifest, graphic url takes normal link to the landing page)

Reply:
thanks @eu-genia. Url is there now.